jhu/params.json.template

```diff
@@ -48,5 +48,10 @@
     "bucket_name": "",
     "cache_dir": "./cache",
     "indicator_prefix": "jhu"
+  },
+  "flash": {
+    "signals": ["confirmed_incidence_num"],
+    "lags": ["1"],
+    "support": ["0", "400000000"]
```

Comment:
nitpick (non-blocking): If these values are being used as numbers, prefer to format them as such (rather than as strings) here for clarity and convenience of not having to convert types in python.

I saw this was a recent change. Did you have some issue with the previous int type?

Reply:
Ditto for local params template.

Reply:
It wouldn't pass lint without the quotes (something about json needs double quotes). Have you seen this issue before?

Reply:
Ah, guess it was something else - thanks for the catch!